src/main/resources/application.properties

```diff
@@ -8,3 +8,5 @@ spring.flyway.password=root
 spring.flyway.locations=classpath:db/migration
 spring.flyway.enabled=true
 spring.flyway.baseline-on-migrate=true
+
+secret=secret
```

Comment:
let's change secret=secret to something like this secret=job_board_secret
or you can made up something by yourself
but this isn't good let it in this way secret=secret.
thank you for good coding.